### PR TITLE
refactor: remove output path logic as it's handled in universal schem…

### PR DESCRIPTION
**Pull request by alan-agius4**
refactor: remove output path logic as it's handled in universal schematic

If the outputPath change is needed, this should be done at upstream in `angular/cli` instead

https://github.com/angular/angular-cli/blob/102cd86a8ea33bf1c3dc8cb75b8727c2f239c3d5/packages/schematics/angular/universal/index.ts#L42

### CODE REVIEW

**Comment by alan-agius4** [comment]
//cc @clydin would you also mind taking a look at this please?

File: modules/common/schematics/install/index.spec.ts [hunk 1]
```diff
@@ -32,15 +32,24 @@ describe('Universal Schematic', () => {
     const contents = JSON.parse(tree.readContent('angular.json'));
     const architect = contents.projects.bar.architect;
     expect(architect.build.configurations.production).toBeDefined();
-    expect(architect.build.options.outputPath).toBe('dist/bar/browser');
-    expect(architect.server.options.outputPath).toBe('dist/bar/server');
+    expect(architect.build.options.outputPath).toBe('dist/bar');
```
Comment (alan-agius4) [inline modules/common/schematics/install/index.spec.ts]: Note: these tests will need to be changed once https://github.com/angular/angular-cli/pull/15717 is landed and released.

Comment (alan-agius4) [inline modules/common/schematics/install/index.spec.ts]: Note: these tests will need to be changed once https://github.com/angular/angular-cli/pull/15717 is landed and released.

Comment (alan-agius4) [inline modules/common/schematics/install/index.spec.ts]: Note: these tests will need to be changed once https://github.com/angular/angular-cli/pull/15717 is landed and released.

**Comment by vikerman** [comment]
We'll wait for the next release of CLI and check  in with adjusted tests so that they don't break out of context.

**Comment by vikerman** [comment]
@alan-agius4 - The next CLI version has been published. Please update the tests. Thanks!

**Comment by alan-agius4** [comment]
@vikerman PR updated

**Comment by CaerusKaru** [comment]
Needs rebase

**Comment by alan-agius4** [comment]
@CaerusKaru, rebased again

**Pull request merged**